packages/webapp/src/_app/ui/delegate-badge.tsx

```diff
@@ -0,0 +1,12 @@
+import { FaStar } from "react-icons/fa";
+
+export const DelegateBadge = ({ size }: { size: number }) => (
```

Comment:
Should we keep our size pattern? eg for buttons:
`export type ButtonSize = "sm" | "md" | "lg";`

Reply:
I don't know...yet. For right now, I see this more like a composed react-icon where size is flexibly variable. But if we come up with a strong convention on badge use and sizing, then yes, I think we make this more opinionated.